src/cunumeric/sort/sort.cu

```diff
@@ -292,23 +315,17 @@ void local_sort(const legate_type_of<CODE>* values_in,
                 int64_t* indices_out,
                 const size_t volume,
                 const size_t sort_dim_size,
-                const bool stable_argsort,  // cub sort is always stable
+                const bool stable,  // cub sort is always stable
                 cudaStream_t stream)
 {
   using VAL = legate_type_of<CODE>;
   // fallback to thrust approach as segmented radix sort is not suited for small segments
-  if (volume == sort_dim_size || sort_dim_size > 300) {
+  if (volume == sort_dim_size || sort_dim_size > 400) {
```

Comment:
Please name the hardcoded integer `400` with some meaningful name.

Reply:
I extracted and documented the threshold.